api/can_predictions.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+import pydantic
+
+"""
+state, 
+countyName, 
+fips, 
+hospitalOverloadDate, 
+hospitalizationPeakDate, 
+aggregateDeaths
+"""
+class _CountyCases(pydantic.BaseModel):
+    fips: str = pydantic.Field(..., description="FIPS state code + FIPS county code")
+    cases: int = pydantic.Field(..., description="Cumulative case count.")
+    deaths: int = pydantic.Field(..., description="Cumulative deaths count.")
+    date: str = pydantic.Field(..., description="Date of latest data")
+
+
+class StateCaseSummary(pydantic.BaseModel):
```

Comment:
I guess I'm not sure how this schema here quite relates to the one in in the top counties pipeline. I feel like the columns should be what's in the docstring above

I would expect a step in the `top_counties_pipeline` to be conforming the data to this.

Reply:
this was still WIP stuff it's all changed now